srcpkgs/libcgroup/template

```diff
@@ -1,18 +1,16 @@
 # Template file for 'libcgroup'
 pkgname=libcgroup
-version=0.41
-revision=8
+version=2.0
+revision=1
 build_style=gnu-configure
-configure_args="--sbindir=/usr/bin --enable-pam-module-dir=/usr/lib/security"
 hostmakedepends="flex automake libtool"
 makedepends="pam-devel"
 short_desc="Library that abstracts the control group file system in Linux"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="LGPL-2.1"
-homepage="http://libcg.sourceforge.net"
-distfiles="${SOURCEFORGE_SITE}/libcg/${pkgname}-${version}.tar.bz2"
-checksum=e4e38bdc7ef70645ce33740ddcca051248d56b53283c0dc6d404e17706f6fb51
-
+homepage="https://github.com/libcgroup/libcgroup"
+distfiles="${homepage}/releases/download/v${version}/${pkgname}-${version}.tar.gz"
+checksum=aecc501a9ea6a97da0673585db5081df912ae607dec36d5f6f7ab14f69d48ab8
 case "$XBPS_TARGET_MACHINE" in
```

Comment:
add an empty line between checksum= and case.

Reply:
added

Reply:
this was not solved

Reply:
this was not solved

sorry for the double comment